docs/zh/samples/eac_runtime.md

```diff
@@ -47,13 +47,13 @@ metadata:
   name: mydemo
 spec:
   mounts:
-    - mountPoint: "eac://nas-mount-point-address:/sub/path"
+    - mountPoint: "efc://nas-mount-point-address:/sub/path"
```

Comment:
I think this prefix should also be modified in the code, otherwise the example yaml may return ["invalid error"](https://github.com/fluid-cloudnative/fluid/blob/master/pkg/ddc/eac/utils.go#L181)

Reply:
fix